### PR TITLE
Added kubeconfig retrieval via clusterctl

### PR DESCRIPTION
**Pull request by kevng9**
Do we want to also include clusterctl steps in the docs as well?

### CODE REVIEW

**Comment by Schnitzel** [comment]
my idea was that the quickstart uses as few tools as possible, having customers to install `clusterctl` just for getting the kubeconfig, while it can easily be loaded via `kubectl` feels strange.
I think however it's ok to add the possiblity to use `clusterctl` to a more in depth explanation of how managedClusters can be created. 
In the quickstart though I would leave it out.

File: docs/quick-start/aws.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ kubectl apply -f aws-cluster-identity-secret.yaml
 
 ## Step 3: Create AWSClusterStaticIdentity Object
 
-Save the AzureClusterIdentity YAML into a file named `aws-cluster-identity.yaml`:
+Save the AWSClusterIdentity YAML into a file named `aws-cluster-identity.yaml`:
```
Comment (Schnitzel) [inline docs/quick-start/aws.md:92]: I do like this though, can you remove the `clusterctl` stuff and leave this in?

**Review by Schnitzel — CHANGES_REQUESTED** [review]

File: docs/quick-start/aws.md (hunk 1 above)
Comment (randybias) [reply docs/quick-start/aws.md:92]: In my incoming PRs I simply put a top level bullet list of "you may want to install these tools:" and listed clusterctl.  Is that OK?

**Comment by randybias** [comment]
@Schnitzel I had a few notes kind of related to this.  I think it would make sense to have a "troubleshooting" document here in the QuickStart guide.  We just put it at the end, but it could include examples of using some of these other tools to help with certain troubleshooting steps.

File: docs/quick-start/aws.md (hunk 1 above)
Comment (randybias) [reply docs/quick-start/aws.md:92]: The above is already fixed in my previous PR.

**Comment by Schnitzel** [comment]
> @Schnitzel I had a few notes kind of related to this. I think it would make sense to have a "troubleshooting" document here in the QuickStart guide. We just put it at the end, but it could include examples of using some of these other tools to help with certain troubleshooting steps.

yea I think a troubleshooting guide makes sense, but not in the quickstart section, as someone that already has passed the quickstart I might not look inside the quickstart section for troubleshooting. I think it should be just in the main navigation

**Comment by kevng9** [comment]
I agree with keeping the quickstart as simple as possible. Troubleshooting guide with other tools makes more sense. Will close this PR

**Pull request closed without merging**